hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/commit/BaseCommitActionExecutor.java

```diff
@@ -238,10 +248,23 @@ protected abstract Iterator<List<WriteStatus>> handleUpdate(String partitionPath
 
   protected HoodieWriteMetadata<HoodieData<WriteStatus>> executeClustering(HoodieClusteringPlan clusteringPlan) {
```

Comment:
Looks this is only fixed inside clustering/delete, is it possible 2 concurrent `insert overwrite` queries also has such issue?

Reply:
Yes, there will be such problems in insert overwrite and compaction. At present, it is only implemented in clustering. After ensuring the correct idea, it will be extended to other actions.Then we can ignore the validation by HUDI-5553